extensions/amp-bind/0.1/validator-amp-bind.protoascii

```diff
@@ -59,6 +59,8 @@ tags: {  # <amp-state> with json child
     name: "id"
     mandatory: true
   }
+  # <amp-bind>
```

Comment:
What if you have some data that can go stale and needs to be refreshed, but you also have some data that won't change? If the child JSON and src attribute are mutually exclusive then all the data, including the data that definitely won't change, will need to be fetched. Since this PR introduces logic for amp-state tags of both kinds to receive fresh data and merge it into existing state, I don't really see the downside in allowing both a src attribute and a child JSON.